src/hooks/useMultiplayerState.ts

```diff
@@ -32,6 +32,7 @@ declare const window: Window & { app: TldrawApp };
 export function useMultiplayerState(
```

Comment:
Can you change this function head to take an object instead of positional arguments.
The last 2 can easily be confused.

Also move openFromFileSystem & updateDoc outside of the function. There are not closures needed from my view to define them inside of useMultiplayerState